src/picongpu/include/plugins/adios/ADIOSWriter.def

```diff
@@ -74,16 +74,15 @@ namespace po = boost::program_options;
     }                                                                         \
 }
 
-#define ADIOS_CMD_EXPECT_NONZERO(_cmd)                                        \
+#define ADIOS_CMD_EXPECT_NONNULL(_cmd)                                        \
 {                                                                             \
-    int _err_code = _cmd;                                                     \
-    if (_err_code == 0)                                                       \
+    if (!(_cmd))                                                              \
```

Comment:
why did you change this part? I think we explicitly wrote it that way to access the `_cmd` var only once.